assignments/todo_list_using_api/script.js

```diff
@@ -0,0 +1,54 @@
+
+var alltodo=document.getElementById("All");
+var pendingtodo=document.getElementById("pend");
+var completedtodo=document.getElementById("done");
+
+
+const api_url = "https://jsonplaceholder.typicode.com/todos";
+async function getapi(url) {
```

Comment:
try adding a filter so that you can show all of them in a group or just completed or incomplete